README.md

```diff
@@ -1,4 +1,9 @@
 nfc
 ===
 
-on behalf of W3C NFC working group
\ No newline at end of file
+on behalf of W3C NFC working group
+
+index.html: the current Editor's draft with separate handling of peers and tags.
+
+index2.html: an experimental version which merges peer and tag read handling,
```

Comment:
But we have two entirely different concepts of API here. It cannot be merged into one doc. If we have to stick with one, it's the current one, which just got through the CfC. The other then cannot be published yet, there is an issue for tracking it, then that's it. What is the problem in having two documents there? We do need both experimental and base versions accessible from the web, i.e. they need to be two different files in the same branch.